posts/Guillaume/2017-07-11-hibernate-validator-600-cr3-out.adoc

```diff
@@ -0,0 +1,41 @@
+= "Hibernate Validator 6.0.0.CR3 is out with Bean Validation 2.0.0.CR3 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+We just released the Bean Validation 2.0 release train (e.g. the specification, the API and the TCK) for the Final Approval Ballot and, as usual, we release a compatible version of Hibernate Validator shortly after: here comes Hibernate Validator 6.0.0.CR3.
```

Comment:
Can you make this a link to the spec announcement (PR pending at https://github.com/beanvalidation/beanvalidation.org/pull/63).